src/ILLink.CodeFix/BaseAttributeCodeFixProvider.cs

```diff
@@ -33,16 +33,22 @@ protected async Task BaseRegisterCodeFixesAsync (CodeFixContext context)
 		{
 			var document = context.Document;
 			var root = await document.GetSyntaxRootAsync (context.CancellationToken).ConfigureAwait (false);
+			if (root is null)
```

Comment:
Nit: what about doing 
```suggestion
			if (await document.GetSyntaxRootAsync (context.CancellationToken).ConfigureAwait (false) is not { } root)
```
To have the same kind of check as the others